integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -1510,6 +1510,30 @@ public static String patchDomainResourceWithNewRestartVersion(
     return String.valueOf(newVersion);
   }
 
+  /**
+   * Patch the domain resource with a new model configMap.
+   *
+   * @param domainResourceName name of the domain resource
+   * @param namespace Kubernetes namespace that the domain is hosted
+   * @param configMapName name of the configMap to be set in spec.configuration.model.configMap
+   */
+  public static void patchDomainResourceWithModelConfigMap(
```

Comment:
can you add a method in Domain.java and call that method from this method, no implementation in TestActions class.

Reply:
Sure. I have also move logic for patchDomainResourceWithModelConfigMap to Domain and call the methods from TestActions. Also did the same to patchDomainResourceWithNewRestartVersion.